### PR TITLE
feat: add tags to map filters

### PR DESCRIPTION
**Pull request by benfurber**
## PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)
- [ ] - Unit and e2e tests for the changes that have been added (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix (fixes an issue)
- [x] Feature (adds functionality)
- [ ] Code style update
- [ ] Refactoring (no functional changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the new behavior?

So far...

https://github.com/user-attachments/assets/c9586170-44a6-41ad-944e-a7cad954f4ad

### CODE REVIEW

File: src/pages/Maps/Maps.client.tsx [hunk 1]
```diff
@@ -96,9 +96,13 @@ const MapsPage = observer(() => {
     ])
   }, [mapPins])
 
+  // Only really for the old map - can be removed soon
   const visibleMapPins = useMemo(() => {
-    return filterMapPinsByType(mapPins, activePinFilters)
-  }, [mapPins, activePinFilters])
+    if (!showNewMap && isPreciousPlastic()) {
```
Comment (mariojsnunes) [inline src/pages/Maps/Maps.client.tsx:101]: I'd rather have
```
const hideMemberPins = !showNewMap && isPreciousPlastic();
return filterMapPinsByType(mapPins, activePinFilters, hideMemberPins);
 ```

**Comment by benfurber** [comment]
@davehakkens It filters out unused tags. I.e. it only shows the filters for the pins that exist. So until a tag is used, it won't be on the list.

File: src/pages/Maps/Maps.client.tsx (hunk 1 above)
Comment (benfurber) [reply src/pages/Maps/Maps.client.tsx:101]: Strong agree.

File: packages/components/src/MapFilterList/MapFilterList.tsx [hunk 2]
```diff
@@ -16,31 +17,43 @@ export interface IProps {
   availableFilters: MapFilterOptionsList
   onClose: () => void
   onFilterChange: (filter: MapFilterOption) => void
+  pinsCount: number
```
Comment (mariojsnunes) [inline packages/components/src/MapFilterList/MapFilterList.tsx:20]: `pinCount` would be more correct, no?

**Review by mariojsnunes — APPROVED** [review]

**Pull request merged**